apps/fluent-tester/src/TestComponents/Text/StandardUsage.tsx

```diff
@@ -4,12 +4,17 @@ import { Text } from '@fluentui/react-native';
 import { Stack } from '@fluentui-react-native/stack';
 import { stackStyle } from '../Common/styles';
 import { TEXT_TESTPAGE } from './consts';
+import { testProps } from '../Common/TestProps';
 
 export const StandardUsage: React.FunctionComponent = () => {
   return (
     <View>
       <Stack style={stackStyle} gap={5}>
-        <Text variant="captionStandard" testID={TEXT_TESTPAGE}>
```

Comment:
I think we can avoid setting a testID here if we pass in a testID in [TextTest.tsx](https://github.com/microsoft/fluentui-react-native/blob/main/apps/fluent-tester/src/TestComponents/Text/TextTest.tsx#L10):

```
  {
    name: 'Standard Usage',
    testID: TEXT_TESTPAGE,
    component: StandardUsage,
  },
```

all the other components follow this pattern except Text

Reply:
and would eliminate the need for `{...testProps(TEXT_TESTPAGE)}` here

Reply:
Added.